### PR TITLE
👌 IMPROVE: Rename the top-level "units"

### LINKED ISSUE #373: 👌 IMPROVE: Tutorial section naming
Currently the sections are named:

- "Fundamentals"
- "Running processes"
- "Data"
- "Workflows"
- "Plugins"

In principle I think it would be better to be more explicit about the difference with "Workchains" and "Running processes" ("aren't workchains a kind of process?") and rename that to "Writting workchains" ("ahh so first you learn to run and then you make your own"). Since we are changing that and I like symmetry, I thought we could adapt the rest as well:

- "Fundamentals" (or "Getting started")
- "Running processes"
- "Managing data"
- "Writing workflows"
- "Creating plugins"

Ideas? Thought? Pizza? 🍕 ?

### PR DESCRIPTION
**Pull request by mbercx**
Fixes #373 

Rename the top-level units (e.g. Data, Workflows, ...) to a more explicit
description of their content (e.g. Managing data, Writing workflows).

Two other minor changes:

* Remove the home page link from the Table of Contents. Users can click the
  tutorial logo to return to the home page.
* Update the year of the copywrite, and remove the reference to Bosch.

### CODE REVIEW

**Comment by mbercx** [comment]
@ramirezfranciscof I've added the `pr/blocked` label because it's probably better to merge other PR's before merging this one. Not sure how `git` handles this, but seems like there is a good chance of merge conflicts popping up all over the place.

**Comment by mbercx** [comment]
@ramirezfranciscof I am seeking your approval, sir! 🧐

**Review by ramirezfranciscof — APPROVED** [review]
Yes, good to go once you fix the problem with the build test!

**Pull request merged**
